userguide/storage.rst

```diff
@@ -1400,8 +1400,8 @@ Find the desired dataset, click |ui-options|, and select
    |                   |                  | or write attributes. Users created manually or imported from a                                             |
    |                   |                  | :ref:`directory service <Directory Services>` appear in the drop-down menu.                                |
    +-------------------+------------------+------------------------------------------------------------------------------------------------------------+
-   | Group             | drop-down menu   | The group which controls the dataset. This group always has permissions to read or write the ACL and       |
-   |                   |                  | read or write attributes. Groups created manually or imported from a                                       |
+   | Group             | drop-down menu   | The group which controls the dataset. This group has whatever permissions are granted to the *@group*      |
```

Comment:
> This group has whatever permissions are granted to the *@group* :guilabel:`Tag`.

The "whatever" seems informal and like you don't know.

Suggestion: `This group has the same permissions that are granted to the *@group* :guilabel:`Tag`.